socioeconomic-differences/socioeconomic-differences.md

```diff
@@ -86,21 +88,20 @@ A person's socioeconomic status can inform their identity, as it affects the com
-
-	2.Students can work together to collect data to find which state in their country saw a major conversion of people form blue-collar job to white-collar job and what were the factors that was responsible, for e.g policies of their local government, increase of jobs due to sudden growth of offices in the areas, reason the offices were setup.  
+	2.Students can work together to collect data to find which state in their country saw a major conversion of people from blue-collar job to white-collar job and what were the factors that was responsible, for e.g policies of their local government, increase of jobs due to sudden growth of offices in the areas, reason the offices were setup.  
    
 
 ### Check for Understanding
 
-
-1. A discussion can be held in the end where the participants summarize each other.
-2. Ask the students to read the first 30 pages of "Blue-Collar Roots, White-Collar Dreams" by Alfred Lubrano,and narrate the essence of what they have read and        understood so far.  
-3. Ask the students to make notes of points mentioned in the book "Hidden Rules of Class at Work" and specifically regarding the problems one faces in order to        achieve promotion as "Class Straddlers".Ask them what they think are the appropriate measures to tackle he mentioned problems.
-4. Give assignment to the students to read " Hillbilly Elegy " by "J.D. Vance" and ask them to explain the problems regarding "work ethics" as narrated in many of      the stories mentioned in the book and ask them to come up with solutions to tackle such issues. 
+    1. A discussion can be held in the end where the participants summarize each other.
+    2. Ask the students to read the first 30 pages of "Blue-Collar Roots, White-Collar Dreams" by Alfred Lubrano,and narrate the essence of what they have read and        understood so far.  
+    3. Ask the students to make notes of points mentioned in the book "Hidden Rules of Class at Work" and specifically regarding the problems one faces in order to        achieve promotion as "Class Straddlers".Ask them what they think are the appropriate measures to tackle the mentioned problems.
+    4. Give assignment to the students to read " Hillbilly Elegy " by "J.D. Vance" and ask them to explain the problems regarding "work ethics" as narrated in many of      the stories mentioned in the book and ask them to come up with solutions to tackle such issues. 
 
 ### Supplemental Materials
-1. https://genprogress.org/straddling-college-and-working-class/ - The following article derives a narrative about the problems working-class students face in       elite schools and furthermore the effect of the problem may hinder their jobs too.
+    
+	1. <https://genprogress.org/straddling-college-and-working-class/> - The following article derives a narrative about the problems working-class students face in       elite schools and furthermore the effect of the problem may hinder their jobs too.
```

Comment:
I suppose '/' isn't required at the end as per [https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)